config/groups.yaml

```
@@ -120,6 +120,11 @@
     - "2666525"
     - "2328859"
   id: 189749
+- name: "seep 3"
+   - "2886851"
+   - "2720012"
+   - "2597043"
+   - "2886997"
```

You're missing the `members` key:
```suggestion
- name: "seep 3"
  members:
    - "2886851"
    - "2720012"
    - "2597043"
    - "2886997"
```